MANUAL.html

```diff
@@ -5143,6 +5143,7 @@ <h2 id="options-74">Options</h2>
       --client-ca string                       Client certificate authority to verify clients with
       --dir-cache-time Duration                Time to cache directory entries for (default 5m0s)
       --dir-perms FileMode                     Directory permissions (default 777)
+      --disable-dir-list                       Disable HTML directory list on GET request for a directory
```

Comment:
Thank you for your contribution. I've just had a quick look at it for now, and first of all you should drop your changes in all three MANUAL files. These are autogenerated (see [this](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#writing-documentation) for details).